packages/lit-html/src/lib/lit-html.ts

```diff
@@ -195,15 +195,15 @@ class Template {
 
     // End of text is: `<` followed by:
     // (comment start) or (tag) or (unescaped <) or (dynamic tag)
-    const textRegex = /<(?:(!--|\/[^a-zA-Z])|(\/?[a-zA-Z]\w*)|(\s)|(\/?$))/g;
+    const textRegex = /<(?:(!--|\/[^a-zA-Z])|(\/?[a-zA-Z][^>\s]*)|\s|(\/?$))/g;
```

Comment:
I don't think we need the `|\s` part at all, since we don't do anything with this kind of match.
```suggestion
    const textRegex = /<(?:(!--|\/[^a-zA-Z])|(\/?[a-zA-Z][^>\s]*)|(\/?$))/g;
```